cfripper/config/rule_configs/firehose_ips.py

```diff
@@ -0,0 +1,56 @@
+from cfripper.config.filter import Filter
+from cfripper.model.enums import RuleMode
+
+"""
+To use this RuleConfig, or any RuleConfig, make sure to include it in the `Config` instantiation.
+
+```python
+RULES_CONFIG = {
+    "EC2SecurityGroupOpenToWorldRule": RuleConfig(
+        filters=[firehose_ips_rules_config_filter]
+    )
+}
+
+config = Config(
+    ...
+    rules_config=RULES_CONFIG,
+)
+```
+"""
+
+# Adapted from https://docs.aws.amazon.com/firehose/latest/dev/controlling-access.html
+FIREHOSE_IPS = [
```

Comment:
Could you use a set, sort them and add a comment on each one with the region name?

Reply:
Done 👍